pkg/assembler/backends/neo4j/src.go

```diff
@@ -549,3 +525,34 @@ func setSrcMatchValues(sb *strings.Builder, src *model.SourceSpec, objectSrc boo
 		}
 	}
 }
+
+func generateModelSource(srcType, namespaceStr, nameStr string, commitValue, tagValue interface{}) model.Source {
+	tag := (*string)(nil)
+	if tagValue != nil {
+		// make sure to take a copy
+		tagStr := tagValue.(string)
+		tag = &tagStr
+	}
+	commit := (*string)(nil)
+	if commitValue != nil {
+		// make sure to take a copy
+		commitStr := commitValue.(string)
+		commit = &commitStr
```

Comment:
No longer need the copy now, it's copied in the method call

Reply:
Would still need to do this to pass it by reference